pat_helper.py

```diff
@@ -198,11 +201,15 @@ def revoke_pat():
     global PAT
     PAT = 'revoked'
 
+
 def print_outputs():
     print(f"::set-output name=username::{USERNAME}")
     print(f"::set-output name=pat::{PAT}")
     print(f"::set-output name=pat_id::{PAT_ID}")
 
+    # STATE_CLEANUP_PAT_ID will be used in the post action phase to automatically revoke the PAT
```

Comment:
some sort of automagic from GHA?

Reply:
Yeah, `::save-state` command would automagically add the `STATE_` prefix